pkg/sql/colexec/expr_eval.go

```diff
@@ -41,6 +47,66 @@ var (
 	constTimestampType  = types.Type{Oid: types.T_timestamp}
 )
 
+func getConstVec(proc *process.Process, expr *plan.Expr, length int) (*vector.Vector, error) {
+	var vec *vector.Vector
+	t := expr.Expr.(*plan.Expr_C)
+	if t.C.GetIsnull() {
+		if types.T(expr.Typ.GetId()) == types.T_any {
+			vec = vector.NewConstNull(types.Type{Oid: types.T(expr.Typ.GetId())}, length)
```

Comment:
I am still not sure what is a T_any.   Is T_any only used to indicate a typeless NULL?

Reply:
select * from t where c1 > ? and c2 < @var 
the ? & var is T_any also.   

and some case I don't know.